sdk/openai/Azure.AI.OpenAI/tests/Samples/05_Images.cs

```diff
@@ -0,0 +1,223 @@
+        ImageClient imageClient = azureClient.GetImageClient("my-dalle-deployment");
+
+        #region Snippet:CreativeImageWorkflow
+        // Creative workflow: Generate marketing materials for a fictional product
+        string[] prompts = {
+            "A sleek smartphone on a minimalist white background, product photography style",
+            "The same smartphone being used by a young professional in a modern office",
+            "A dramatic artistic shot of the smartphone with colorful lighting effects"
+        };
+
+        string[] contexts = { "product", "lifestyle", "artistic" };
+
+        Console.WriteLine("Generating marketing image set...");
+        
+        for (int i = 0; i < prompts.Length; i++)
+        {
+            ImageGenerationOptions options = new()
+            {
+                Size = GeneratedImageSize.W1024xH1024,
+                Quality = GeneratedImageQuality.Standard,
+                Style = i == 2 ? GeneratedImageStyle.Vivid : GeneratedImageStyle.Natural,
+                ResponseFormat = GeneratedImageFormat.Bytes,
+            };
+
+            GeneratedImage image = imageClient.GenerateImage(prompts[i], options);
+            
+            // Save each image with descriptive naming
+            byte[] imageBytes = image.ImageBytes.ToArray();
+            string filename = $"marketing_{contexts[i]}_{DateTime.Now:yyyyMMdd_HHmmss}.png";
+            File.WriteAllBytes(filename, imageBytes);
+            
+            Console.WriteLine($"Generated {contexts[i]} image: {filename}");
+            Console.WriteLine($"  Revised prompt: {image.RevisedPrompt}");
+            Console.WriteLine();
+        }
+        
+        Console.WriteLine("Marketing image set complete! Use these images for:");
+        Console.WriteLine("- Product catalog (product image)");
+        Console.WriteLine("- Website hero section (lifestyle image)");
+        Console.WriteLine("- Social media campaigns (artistic image)");
+        #endregion
+    }
+
+    public void ImageGenerationBestPractices()
+    {
+        AzureOpenAIClient azureClient = new(
+            new Uri("https://your-azure-openai-resource.com"),
+            new DefaultAzureCredential());
+        ImageClient imageClient = azureClient.GetImageClient("my-dalle-deployment");
+
+        #region Snippet:ImageGenerationBestPractices
+        // Best practices for prompt engineering and image generation
+        
+        // 1. Detailed and specific prompts yield better results
+        string detailedPrompt = "A macro photograph of a dewdrop on a green leaf, " +
+                               "shot with shallow depth of field, golden hour lighting, " +
+                               "nature photography, high resolution, crisp details";
+
+        // 2. Use appropriate quality settings for your use case
+        ImageGenerationOptions highQualityOptions = new()
+        {
+            Quality = GeneratedImageQuality.Standard, // Use Standard for most applications
```

Comment:
This one is called `highQualityOptions` but it is using `GeneratedImageQuality.Standard`, which feels a little contradictory.